test/README.md

```diff
@@ -131,36 +131,80 @@ You can set these environment variables to configure the test suite:
 See the [bots documentation](https://github.com/cockpit-project/bots/blob/main/README.md)
 for details about the tools and configuration for these.
 
-## Faster iteration
+## Convenient test VM SSH access
+
+It is recommended to add a snippet like this to your `~/.ssh/config`. Then
+you can log in to test machines without authentication:
+
+    Match final host 127.0.0.2
+        User root
+        StrictHostKeyChecking no
+        UserKnownHostsFile /dev/null
+        CheckHostIp no
+        IdentityFile ~/src/cockpit/bots/machine/identity
```

Comment:
Shall we maybe add a hint that this path needs to be replaced. Mostly all gsoc contributors copy pasted this as is - and where asking then on matrix why ssh c does not work :)

Reply:
Good point, thanks for catching! I added another commit for that.